app/routes/endemics/timings.js

```diff
@@ -0,0 +1,42 @@
+const Joi = require('joi')
+const urlPrefix = require('../../config/index').urlPrefix
+const {
+  endemicsNumbers,
+  endemicsTimings,
+  endemicsOfferRejected,
+  endemicsDeclaration
+} = require('../../config/routes')
+
+const backLink = `${urlPrefix}/${endemicsNumbers}`
+
+module.exports = [
+  {
+    method: 'GET',
+    path: `${urlPrefix}/${endemicsTimings}`,
+    options: {
+      handler: async (request, h) => {
+        return h.view('endemics/timings', {
+          backLink
+        })
+      }
+    }
+  },
+  {
+    method: 'POST',
+    path: `${urlPrefix}/${endemicsTimings}`,
+    options: {
+      validate: {
+        payload: Joi.object({
+          agreementStatus: Joi.string().required().valid('agree', 'rejected')
+        })
+      },
+      handler: async (request, h) => {
+        // session.setFarmerApplyData(request, type, 'EE')
```

Comment:
I believe this line is not required here, and it's been added by Alex in his PR.